UsageStatsLoader.inc.php

```diff
@@ -115,8 +112,8 @@ protected function executeActions() {
 			return false;
 		}
 
-		if (!$this->getCounterRobotListFile() || !file_exists($this->getCounterRobotListFile())) {
-			$this->addExecutionLogEntry(__('plugins.generic.usageStats.noCounterBotList', array('botlist' => $this->getCounterRobotListFile())), SCHEDULED_TASK_MESSAGE_TYPE_WARNING);
+		if (!file_exists(COUNTER_USER_AGENTS_FILE)) {
```

Comment:
I'd suggest removing this check entirely (and the locale key, if there's no other use for it). No sense checking for missing files when they're built into the distribution, I think.